lib/house.rb

```diff
@@ -0,0 +1,35 @@
+class House
+
+  attr_reader :price, :address, :rooms
+  def initialize(price, address)
+    @price = price
```

Comment:
If you look closely at the interaction pattern, price should be an integer. so you'd want to do `@price = price.to_i`